### ISSUE TITLE
Test & improve for WCAG 2.0 Level AA Standard

### ISSUE THREAD

**Issue by blutz** [blutz/ninteen-thirty-four#40]
https://www.w3.org/WAI/WCAG21/quickref/

https://www.w3.org/WAI/test-evaluate/

Current violations are just decorative images that don't have alt text ([empty alt should be added](https://www.w3.org/WAI/tutorials/images/decorative/))
